### PR TITLE
Added RisingPixels.net

### PR DESCRIPTION
**Pull request by shloooo**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by shloooo** [comment]
> _Just as an information_: Your server play.risingpixels.net **could be offline**. In general, we only accept pull requests from servers, **that are online**. Please change this, otherwise we cannot review your server correctly and have to deny the pull request.
> 
> If your server is currently online, then our api returned a wrong status, we will have a look at it :)
> 
> Reference: [API URL (https://api.mcsrvstat.us/2/play.risingpixels.net)](https://api.mcsrvstat.us/2/play.risingpixels.net)

Missing srv record were added :)

File: minecraft_servers/risingpixels/manifest.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "server_name": "risingpixels",
+  "nice_name": "RisingPixels",
+  "direct_ip": "play.risingpixels.net",
+  "server_wildcards": [
+    "%.risingpixels.net"
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:6]: Not a real wildcard
https://i.imgur.com/pc021dx.png

File: minecraft_servers/risingpixels/manifest.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://risingpixels.net",
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:13]: 403 error

File: minecraft_servers/risingpixels/manifest.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "web_shop": "https://store.risingpixels.net",
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:14]: Not reachable

File: minecraft_servers/risingpixels/manifest.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "web_support": "https://discord.gg/aHvRGg4hqb",
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:15]: Web_support cannot be Discord, either contact form or FAQ
If nonexistent, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by shloooo** [comment]
Should work now @DoJapHD :)

Shop were removed and support page link were updated :)

File: minecraft_servers/risingpixels/manifest.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+  "server_name": "risingpixels",
+  "nice_name": "RisingPixels",
+  "direct_ip": "risingpixels.net",
+  "server_wildcards": [
+    "player.risingpixels.net"
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:6]: That's not a wildcard
https://en.wikipedia.org/wiki/Wildcard_DNS_record

File: minecraft_servers/risingpixels/manifest.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://www.risingpixels.net",
+    "web_support": "https://www.risinglife.net/support",
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:14]: Not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/risingpixels/manifest.json (hunk 5 above)
Comment (shloooo) [reply minecraft_servers/risingpixels/manifest.json]: Removed wildcards now :)

File: minecraft_servers/risingpixels/manifest.json (hunk 6 above)
Comment (shloooo) [reply minecraft_servers/risingpixels/manifest.json]: Fixed wrong domain xd

File: minecraft_servers/risingpixels/manifest.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
+  "server_name": "risingpixels",
+  "nice_name": "RisingPixels",
+  "direct_ip": "risingpixels.net",
+  "server_wildcards": [],
```
Comment (DoJapHD) [inline minecraft_servers/risingpixels/manifest.json:5]: If nonexistent, remove entirely

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/risingpixels/manifest.json (hunk 7 above)
Comment (shloooo) [reply minecraft_servers/risingpixels/manifest.json]: 👍

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**